docs/source/docs/index.rst

```diff
@@ -256,7 +256,7 @@ Read the research:
    ../examples/managed-jobs
    Multi-Node Jobs <../running-jobs/distributed-jobs>
    Many Parallel Jobs <../running-jobs/many-jobs>
-
+   Model Training Guide <../reference/training-guide>
```

Comment:
Add a `new` badge? Can be done by editing the `custom.js`: https://github.com/KeplerC/skypilot/blob/344ecb862bdde52331779be34661641fb31441a3/docs/source/_static/custom.js#L27-L35